networking/eth2/src/testFixtures/java/tech/pegasys/teku/networking/eth2/peers/RespondingEth2Peer.java

```diff
@@ -436,67 +485,97 @@ public void complete() {
     }
   }
 
-  private interface PendingBlockRequestHandler<T> {
+  private interface PendingRequestHandler<T, S> {
```

Comment:
Takes a bit to understand what is it for, especially as S is now for a type and T for response future if I'm getting it right. I'd use long names like in `sendRequest` or at least something like RT, T (though looks bad too)

Reply:
Makes sense, changed it to `ResponseT` and `HandlerT`